secrethub/resource_service_aws.go

```diff
@@ -0,0 +1,72 @@
+package secrethub
+
+import (
+	"fmt"
+
+	"github.com/secrethub/secrethub-go/pkg/secrethub/credentials"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceServiceAWS() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceServiceAWSCreate,
+		Read:   resourceServiceRead,
+		Delete: resourceServiceDelete,
+		Schema: map[string]*schema.Schema{
+			"repo": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The path of the repository on which the service operates.",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "A description of the service so others will recognize it.",
+			},
+			"kms_key": {
```

Comment:
`kms_key_arn` I'd say, if we require it to be an ARN.

Reply:
Yes, that might be clearer.

If we do so, should we also use `repo_path` instead of `repo`?

Reply:
Would we support repo ids later on?

Reply:
Yes, we might.

Could we support KMS keys in another format later?

Reply:
We wanted the KMS ARN to get the enclosed region.

Reply:
Discussed this in person and concluded that the KMS key field is not likely to accept anything other than the ARN and that it might be confusing that it does not accept e.g. the KMS id when the name is only `kms_key`. Therefore, we'll rename the field `kms_key_arn`.